src/amo/components/Errors/AuthExpired/styles.scss

```diff
@@ -0,0 +1,7 @@
+@import '~amo/css/styles';
+
+.AuthExpired .Button {
+  color: $blue-50;
```

Comment:
Do we perhaps have a constant that we use for link-color, or something like that?

Reply:
I searched for `text-decoration: underline;` to try to find something similar and the matches were mixed - two of them used `$blue-50` so I went with that.  One does use `$link-color` I now see (among others that use `$white` and `$black`).  I'll change it to `$link-color`